src/ImportDialog.vala

```diff
@@ -41,13 +41,13 @@ public class Maya.View.ImportDialog : Granite.MessageDialog {
                 critical (e.message);
             }
 
-            secondary_text = _("Events from \"%s\" will be merged with this calendar:").printf (name);
+            secondary_text = _(("Events from \"%s\" will be merged with this calendar:").printf (name));
```

Comment:
This is wrong, the text has to be translated first then replaced

Reply:
You're right, sorry. I've found the problem, someone mistranslated it in Italian using %d instead of %s... 🤦🏻‍♂️️